File: docs/fundamentals/runtime-libraries/snippets/System/Double/Overview/vb/precisionlist1.vb
' Visual Basic .NET Document
Option Strict On

' <Snippet5>
Module Example10
    Public Sub Main()
        Dim value1 As Double = 1 / 3
        Dim sValue2 As Single = 1 / 3
        Dim value2 As Double = CDbl(sValue2)
        Console.WriteLine("{0} = {1}: {2}", value1, value2, value1.Equals(value2))
    End Sub
End Module
' The example displays the following output:
'       0.33333333333333331 = 0.3333333432674408: False
' </Snippet5>

File: docs/fundamentals/runtime-libraries/snippets/System/Double/Overview/csharp/precisionlist1.cs
// <Snippet5>
using System;

public class Example9
{
    public static void Main()
    {
        Double value1 = 1 / 3.0;
        Single sValue2 = 1 / 3.0f;
        Double value2 = (Double)sValue2;
        Console.WriteLine($"{value1:R} = {value2:R}: {value1.Equals(value2)}");
    }
}
// The example displays the following output:
//        0.33333333333333331 = 0.3333333432674408: False
// </Snippet5>
